commit c9c5208f77d560467965619fadbf350ada9a0bc2
Author: Christian Boltz <apparmor@cboltz.de>
Date:   Mon Aug 8 20:48:12 2022 +0200

    dnsmasq: Add missing r permissions for libvirt_leaseshelper
    
    Note: This was reported for /usr/libexec/libvirt_leaseshelper, but since
    this is probably unrelated to the path or a path change, this commit
    also adds r permissions for the previous path.
    
    Fixes: https://bugzilla.opensuse.org/show_bug.cgi?id=1202161

diff --git a/profiles/apparmor.d/usr.sbin.dnsmasq b/profiles/apparmor.d/usr.sbin.dnsmasq
index bffc09b4..406b2599 100644
--- a/profiles/apparmor.d/usr.sbin.dnsmasq
+++ b/profiles/apparmor.d/usr.sbin.dnsmasq
@@ -117,8 +117,8 @@ profile dnsmasq /usr/{bin,sbin}/dnsmasq flags=(attach_disconnected) {
 
     /etc/libnl-3/classid r,
 
-    /usr/lib{,64}/libvirt/libvirt_leaseshelper m,
-    /usr/libexec/libvirt_leaseshelper m,
+    /usr/lib{,64}/libvirt/libvirt_leaseshelper mr,
+    /usr/libexec/libvirt_leaseshelper mr,
 
     owner @{PROC}/@{pid}/net/psched r,
     owner @{PROC}/@{pid}/status r,
